test/ae_mdw_web/controllers/block_controller_test.exs

```diff
@@ -12,47 +13,40 @@ defmodule AeMdwWeb.BlockControllerTest do
 
   import AeMdw.Db.Util
   import Mock
+  import AeMdwWeb.BlockchainSim
 
   @blocks_table AeMdwWeb.BlockController.table()
   @default_limit 10
   @blocks_cache_threshold 6
 
   describe "block" do
     test "get key block by hash", %{conn: conn} do
-      kb_hash = "kh_29Gmo8RMdCD5aJ1UUrKd6Kx2c3tvHQu82HKsnVhbprmQnFy5bn"
-      bin_kb_hash = AeMdw.Validate.id!(kb_hash)
+      with_blockchain %{alice: 10_000}, [{:b1, {:kb, :alice}}] do
+        kb_hash = blocks[:b1]
 
-      with_mocks [
-        {:aec_chain, [], get_block: fn ^bin_kb_hash -> {:ok, sample_key_block()} end},
-        {:aec_db, [], get_header: fn ^bin_kb_hash -> sample_key_header() end}
-      ] do
-        conn = get(conn, "/block/#{kb_hash}")
-
-        assert json_response(conn, 200) == TestUtil.handle_input(fn -> get_block(kb_hash) end)
+        assert %{"hash" => ^kb_hash} = get(conn, "/block/#{kb_hash}") |> json_response(200)
       end
     end
 
     test "get micro block by hash", %{conn: conn} do
-      mb_hash = "mh_RuKG8scokoAdhqNN3uvq29VZBsSaZdpeaoyBsXLeGV69sud85"
-      bin_mb_hash = AeMdw.Validate.id!(mb_hash)
-
-      with_mocks [
-        {:aec_chain, [], get_block: fn ^bin_mb_hash -> {:ok, sample_micro_block()} end},
-        {:aec_db, [], get_header: fn ^bin_mb_hash -> sample_micro_header() end}
+      with_blockchain %{alice: 10_000, bob: 20_000}, [
+        {:b1,
+         {:mb,
+          [
+            {:t1, BlockchainSim.spend_tx(:alice, :bob, 5_000)}
+          ]}}
```

Comment:
Very nice shorter convention on :b1, :b2, :anyatom being accessible by blocks variable. 
What you think about auto declaring a microblock, for an hypothesis of most cases the transaction going into the same microblock:
```
 {:b1,  
  [
    {:t1, BlockchainSim.spend_tx(:alice, :bob, 5_000)}
  ]
}
```
The current proposal is powerful because allows multiple micro blocks but for short, when application, we would assume every transaction would be related to 1 microblock.

Reply:
allows something like:        
```
      with_blockchain %{alice: 10_000, bob: 20_000},
        b1: [
          t1: BlockchainSim.spend_tx(:alice, :bob, 5_000)
        ]
      do
```

Reply:
Good idea, done.